### DISCUSSION TITLE
Extract timings from program outputs

### DISCUSSION THREAD

**Discussion by timfi** [sharkdp/hyperfine#743] [Ideas]
I often stumble upon wanting to measure runtimes in systems like PostgreSQL or DuckDB, both of which have considerable overhead that is not pertinent to the actual measurement at hand. These two system, and many more, have their own timing features, which usually just dump the relevant information into stdout or stderr. Are there any thoughts around enabling hyperfine to read this information rather than timing the whole program execution itself?

**Comment by sharkdp** [comment]
Please see https://github.com/sharkdp/hyperfine/issues/170 and similar tickets.
